instrumentation/external-annotations/external-annotations.gradle

```diff
@@ -1,5 +1,7 @@
 apply from: "$rootDir/gradle/instrumentation.gradle"
 
+// TODO(anuraaga): Reenable after SDK 0.10.0 is published
```

Comment:
I guess this is better than having muzzle pull in random versions from snapshot repo? Let me know if there's a better way.

I'm surprised the smoke tests pass though - actually I had flaky behavior where sometimes they fail with no WithSpan and sometimes they pass fine locally, and have similar seemingly bad-cache behavior before so a bit anxious they may have some problem @iNikem

Reply:
Did something change in the annotations that broke muzzle's compatibility check?

Reply:
Now the implementation uses `Span.makeCurrent()` instead of `TracingContextUtils` which isn't available in previous opentelemetry API